tasks/meta.py

```diff
@@ -511,7 +589,11 @@ class OBSColumn(Base):
                                # these together across geoms: AVG, SUM etc.
 
     tables = relationship("OBSColumnTable", back_populates="column", cascade="all,delete")
-    tags = association_proxy('column_column_tags', 'tag', creator=tag_creator)
+    session = current_session()
+    # Passing the same session object to the creator saves thousands
+    # of `select 1` queries, because each use of the session was pinging
+    tag_creator_with_session = lambda x, session=session: tag_creator(x, session)
```

Comment:
The better way to do this is with [`functools.partial`](https://docs.python.org/3/library/functools.html#functools.partial). Something like `tag_creator_with_session = functools.partial(tag_creator, session=session)`